vignettes/articles/cookbook_preparatory_steps.Rmd

```diff
@@ -68,6 +70,8 @@ pacta.loanbook:::list_col_names_and_types(scenario_demo_2020)
 
 For details on the required structure of this dataset, refer to the [market share scenario data dictionary](data_scenario.html#climate-transition-scenario-dataset-technology-pathways-for-target-market-share-calculation).
 
+#### Sectors with Sector-Level Pathways
+
 For sectors without technology-level pathways, PACTA applies the Sectoral Decarbonization Approach (SDA). This method requires all companies in a sector to align their physical emission intensity with a future scenario target (e.g., by 2050). Higher-emission companies must reduce their intensity more drastically than those already using cleaner technologies. However, SDA does not directly affect production volumes.
 
 A more detailed explanation of the sectoral decarbonization approach can be found [here](cookbook_metrics.html)
```

Comment:
I know you didn't touch this line, but linking directly to the SDA approach documentation here seems relevant

```suggestion
A more detailed explanation of the sectoral decarbonization approach can be found [here](cookbook_metrics.html#sectoral-decarbonization-approach)
```